mock/elasticProcessorStub.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // ElasticProcessorStub -
 type ElasticProcessorStub struct {
-	SaveHeaderCalled                 func(header coreData.HeaderHandler, signersIndexes []uint64, body *block.Body, notarizedHeadersHashes []string, txsSize int) error
+	SaveHeaderCalled                 func(header coreData.HeaderHandler, signersIndexes []uint64, body *block.Body, notarizedHeadersHashes []string, gasConsumptionData indexer.HeaderGasConsumption, txsSize int) error
```

Comment:
you will change the interface. Should change all implementations

Reply:
what do you mean? this interface is elastic-indexer-go specific. I've changed each stub and implementation

Reply:
right, the ElasticProcessor is just a subcomponent used in the indexer implementation and in one integration test in elrond-go